packages/flutter/test/material/search_test.dart

```diff
@@ -1047,6 +1068,11 @@ void main() {
     await tester.pumpAndSettle();
     await tester.tap(find.byTooltip('Close'));
     await tester.pumpAndSettle();
+
+    // showSearch without back button
```

Comment:
This should probably be its own test that verifies the behavior when `automaticallyImplyLeading` is `false`.

Reply:
the test has been added, and the value has been checked. thank you